cmake/knime/binaries_lnx.ini

```diff
@@ -1,7 +1,9 @@
 OPENMS_DATA_PATH=$ROOT/share/OpenMS
+PATH=$ROOT/bin/:$ROOT/bin/Fido:$ROOT/bin/Comet:$ROOT/bin/OMSSA:$ROOT/bin/MyriMatch:$ROOT/bin/XTandem:$ROOT/bin/MaRaCluster:$ROOT/../../../jre/bin/java:${PATH}
 PATH=$ROOT/bin/:$ROOT/bin/Fido:$ROOT/bin/Comet:$ROOT/bin/OMSSA:$ROOT/bin/MyriMatch:$ROOT/bin/XTandem:$ROOT/bin/Percolator:$ROOT/bin/SpectraST:$ROOT/bin/crux:$ROOT/../../../jre/bin/java:${PATH}
 LD_LIBRARY_PATH=$ROOT/lib/:${LD_LIBRARY_PATH}
 MSGFPLUS_PATH=$ROOT/bin/MSGFPlus/MSGFPlus.jar
 LUCIPHOR_PATH=$ROOT/bin/LuciPHOr2/luciphor2.jar
 SIRIUS_PATH=$ROOT/bin/Sirius/sirius
+MARACLUSTER_PATH=$ROOT/bin/MaRaCluster/maracluster
```

Comment:
I think you only need this if it is a jar or started by a shell script  that needs that variable.